lib/widgets/fetchErrorHandler.dart

```diff
@@ -0,0 +1,124 @@
+// ignore: file_names
+import 'package:trufi_core/blocs/app_review_cubit.dart';
+import 'package:trufi_core/blocs/home_page_cubit.dart';
+import 'package:trufi_core/blocs/location_provider_cubit.dart';
+import 'package:trufi_core/blocs/preferences_cubit.dart';
+import 'package:trufi_core/l10n/trufi_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:trufi_core/repository/exception/fetch_online_exception.dart';
+import 'package:trufi_core/trufi_configuration.dart';
+import 'package:url_launcher/url_launcher.dart';
+// ignore: implementation_imports
```

Comment:
What is this about?

Reply:
![Screen Shot 2021-04-29 at 04 02 58](https://user-images.githubusercontent.com/43000324/116519716-d151d680-a89f-11eb-932b-c209311ae326.png)

I do not know how to fixed it

Reply:
https://dart-lang.github.io/linter/lints/implementation_imports.html
Actually it is a really funny issue:
just replace line 11 with `import 'package:provider/provider.dart';`

You are not allowed to import internals of the package.